### DISCUSSION TITLE
Functional form of SMIRNOFF pair switch

### DISCUSSION THREAD

**Discussion by Jkmet** [openforcefield/discussions#38] [Q&A]
I am considering simulating a large number of polymer systems in LAMMPS with OpenFF. The OpenFF/SMIRNOFF documentation discusses pair switching, but I cannot find the functional form of said switch, only that it is by default applied from 8 to 9 angstroms. LAMMPS has a variety of options for pair switching options and documentation on their functional forms, so I expect an appropriate switch exists in LAMMPS.

For example, here is documentation on the function forms of a subset of LAMMPS switches:
https://docs.lammps.org/Howto_bioFF.html

**Comment by mattwthompson** [comment]
The functional form is whatever OpenMM happens to use, which I believe is documented here: http://docs.openmm.org/latest/userguide/theory/02_standard_forces.html?highlight=switching%20function#lennard-jones-interaction

The last time I looked into this, it wasn't obvious how to implement this in LAMMPS: 
https://github.com/openforcefield/openff-interchange/issues/159

Happy to take a PR that implements this in LAMMPS
